segments/now_playing.sh

```diff
@@ -121,6 +125,9 @@ __process_settings() {
 	if [ -z "$TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR" ]; then
 		export TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR="${TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR_DEFAULT}"
 	fi
+	if [ -z "$TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT" ]; then
+		export TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT="${TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT}"
```

Comment:
The value should be `TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT_DEFAULT` right?

I think it's the same issue on `TMUX_POWERLINE_SEG_NOW_PLAYING_RHYTHMBOX_FORMAT` below as well. Maybe you can fix it too? :)

Reply:
Ah, yes this is a copy-and-paste mistake :innocent: 
Both is fixed in the latest commit.